src/chart/custom/CustomView.ts

```diff
@@ -1316,24 +1322,52 @@ function mergeChildren(
     // might be better performance.
     let index = 0;
     for (; index < newLen; index++) {
-        newChildren[index] && doCreateOrUpdateEl(
-            api,
-            el.childAt(index),
-            dataIndex,
-            newChildren[index] as CustomElementOption,
-            seriesModel,
-            el
-        );
+        const newChild = newChildren[index];
+        const oldChild = el.childAt(index);
+        if (newChild) {
+            if (newChild.ignore == null) {
```

Comment:
Should be:
```ts
if (newChild.ignore) {
    newChild.ignore = false;
}
```
or 
```ts
newChild.ignore = null; // or false
```

Reply:
`newChild` is the user-provided option. So if `newChild.ignore` is true, it means the user want to ignore the element so I don't think we should set `newChild.ignore = false` in this situation.

Reply:
I got it wrong

Reply:
What if users set a child as `null` / `undefined` at the first time ? (rather than the second time)